cmake/Bond.cmake

```diff
@@ -129,6 +130,18 @@ function (add_bond_test test)
             WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
             COMMAND ${test})
     endif()
+
+    get_property (lastTestPort GLOBAL PROPERTY LAST_TEST_PORT)
+    if (NOT DEFINED lastTestPort)
+        set (lastTestPort 25189)
+    endif()
+    math(EXPR testPort1 "${lastTestPort} + 1")
+    math(EXPR testPort2 "${lastTestPort} + 2")
+    target_compile_definitions (${test} PRIVATE -DTEST_PORT_1=${testPort1})
```

Comment:
Fun fact: `target_compiler_definition` handles [both](https://cmake.org/gitweb?p=cmake.git;a=blob;f=Source/cmTargetCompileDefinitionsCommand.cxx;h=008d1a210a9ec05fe9310794576d47267e68f46a;hb=HEAD#l46) `MACRO` and `-DMACRO`.